src/contentful/contentful.service.ts

```diff
@@ -25,20 +25,21 @@ const CALLS = 8
 
 @Injectable()
 export class ContentfulService {
-  async getAllEntries(): Promise<Entry[]> {
-    const allEntryCollections: EntryCollection<EntrySkeletonType>[] = []
-
+  async getAllEntries() {
+    const allEntryCollections: EntryCollection<EntrySkeletonType>[] = [];
     for (let i = 0; i < CALLS; i++) {
       const entryCollection = await contentfulClient.getEntries({
         limit: LIMIT,
+        include: 5,
         skip: i * LIMIT,
-      })
-      allEntryCollections.push(entryCollection)
+      });
+      allEntryCollections.push(entryCollection);
     }
-
-    return allEntryCollections.reduce((acc, entryCollection) => {
+    const allEntries = allEntryCollections.reduce((acc, entryCollection) => {
       return [...acc, ...entryCollection.items]
     }, [] as Entry[])
+
+    return allEntries;
```

Comment:
Why the a redundant `allEntries` variable addition here? 🤔

Reply:
Was for debugging, I was printing it. I'll change it back